user/serializers.py

```diff
@@ -22,3 +30,43 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        label=_("Email"),
+        write_only=True
+    )
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True
+    )
+    token = serializers.CharField(
+        label=_("Token"),
+        read_only=True
+    )
+
+    def validate(self, attrs):
+        email = attrs.get("username")
```

Comment:
The `validate` method in `AuthTokenSerializer` incorrectly retrieves the email using `attrs.get("username")`. Since the field is named `email`, it should be `attrs.get("email")` instead.